doc/developers-guide/domain_management.md

```diff
@@ -156,13 +155,9 @@ curl -v -X DELETE "http://localhost:8088/api/domains/example.db" \
 
 Result codes:
 
-* 204 - the domain is removed or not found.
-* 403 - the domain is static.
-* 403 - the DB service is disabled.
-* 403 - the host type is wrong (does not match the host type in the database).
-* 403 - the host type is unknown.
-* 500 - other errors.
-
+* 204 - The domain is removed or not found.
+* 403 - The domain is static, the DB service is disabled, the host type is wrong (does not match the host type in the database), or the host type is unknown.
```

Comment:
https://github.com/esl/MongooseIM/blob/master/doc/rest-api/Dynamic-domains_swagger.yml#L88 - `Domain is static, or the service is disabled.`

Reply:
* 403 - One of:
  * the domain is static.
  * the DB service is disabled.
  * the host type is wrong (does not match the host type in the database).
  * the host type is unknown.
  
  --->
  
```
* 403 - One of:
  * the domain is static.
  * the DB service is disabled.
  * the host type is wrong (does not match the host type in the database).
  * the host type is unknown.
```

Reply:
Do you mean it lacks the "DB" word? The methods have (almost) the same error descriptions respectively (this is DELETE, the link is to PATCH). The wording is taken from the original form of `domain_management.md`, but a "DB" can be easily added if this is what you have in mind.

Reply:
The command was updated with examples